app/modules/vm/view.jade

```diff
@@ -74,6 +74,7 @@
           dt CPU Weight
           dd
             span(
+              e-ng-disabled="VM.other['xo:resource_set'] && user.permission !== 'admin'"
```

Comment:
Would not it be better to have a dedicated function (`canSetCpuWeight()`).

It's event better because you don't have to inject the user in the scope (and keep referencing `xoApi.user`).